lang-ca.js

```diff
@@ -924,9 +930,11 @@ SnapTranslator.dict.ca = {
     'add comment':
         'afegeix un comentari',
     'undrop':
-        'recupera bloc',
+        'desfés',
     'undo the last\nblock drop\nin this pane':
-        'recupera l\'últim bloc\nque s\'hagi llençat',
+        'desfés l\'últim moviment\nde blocs',
+    'redrop':
+        'refés',
```

Comment:
Yeah, we're going to exchange my new "undrop" mechanism with @brollb 's "real undo" once it's ready for prime time. But since @brollb 's mechanism is much more refined I wanted to make it clear that my hacky kludgy one isn't yet the real thing by choosing a different name :-)